internal/bookmarks/manager.go

```diff
@@ -30,10 +30,24 @@ func (m *Manager) Get(w http.ResponseWriter, r *http.Request) {
 	userId := r.URL.Query().Get("user_id")
 
 	if userId != "" {
-		iUserId, _ := strconv.Atoi(userId)
-		dbBookmarks = m.DbClient.GetBookmarksByUserID(iUserId)
+		iUserId, err := strconv.Atoi(userId)
+		if (err != nil) {
+			log.Printf("%v", err)
+			writeStatus(w, http.StatusBadRequest)
```

Comment:
This is a case where we can update the flow to provide an error message json body response